### ISSUE TITLE
How to clear formatting of the cells while exporting?

### ISSUE THREAD

**Issue by hussaintamboli** [tarkatronic/django-excel-response#16]
I have a functionality where user exports the data, make changes and then imports the data. This data has and identifier which is stored in db as charfield but is all numbers.

    200420340204032403204

When user exports the file he sees it as

    2.0042E+20

Now when he updates the other cells, and imports the file I read it as 2.0042E+20 as string. I can't find it in my db, so I can't update it.

Is there any way I can clear cell formatting so that I can get the content as is while exporting?

**Comment by hussaintamboli** [comment]
After going through your code, I think I know what needs to be done. I guess when `force_csv` is `False` I need a way to set `workbook.guess_types` to `False`

**Comment by tarkatronic** [comment]
I will see what I can do about this. Thank you for the bug report!

**Comment by tarkatronic** [comment]
@hussaintamboli I've created a test which should reproduce this bug, but I'm not seeing an issue. One theory I have for why you are seeing this is actually that Excel might be performing this truncation for you. It tends to try to be "helpful" like this.

CSV itself is just pure text output, so it does no formatting and has no type hinting for Excel.

Can you try opening the exported CSV file in a plain text editor and see if you experience the same results? Thank you!

And here is the test I added, in case you have any additional thoughts on one:

```python
    def test_csv_output_doesnt_truncate_long_integers(self):
        TestModel.objects.create(text='200420340204032403204', number=1)
        r = response.ExcelResponse(
            TestModel.objects.all().values('text'),
            force_csv=True
        )
        output = r.getvalue()
        self.assertEqual(
            output,
            b'text\r\n200420340204032403204\r\n'
        )
```

**Comment by hussaintamboli** [comment]
I have raised a pull request https://github.com/tarkatronic/django-excel-response/pull/22. Could you please have a look at it?

**Comment by tarkatronic** [comment]
Fixed by #22
